packages/react/src/primitives/PhoneNumberField/CountryCodeSelect.tsx

```diff
@@ -7,20 +7,21 @@ import { CountryCodeSelectProps, Primitive } from '../types';
 import { SelectField } from '../SelectField';
 
 const CountryCodeSelectPrimitive: Primitive<CountryCodeSelectProps, 'select'> =
-  ({ className, dialCodeList, ...props }, ref) => {
+  ({ className, dialCodeList, isReadOnly, ...props }, ref) => {
     const dialList = dialCodeList ?? countryDialCodes;
     const countryCodeOptions = React.useMemo(
       () =>
         dialList.map((dialCode) => (
-          <option key={dialCode} value={dialCode}>
+          <option key={dialCode} value={dialCode} disabled={isReadOnly}>
```

Comment:
Should we add a comment of why we are mapping disabled to isReadOnly? We will wonder later on.